en/docs/get-started/identity-provisioning-and-its-standards.md

```diff
@@ -0,0 +1,190 @@
+# Identity Provisioning and its Standards
```

Comment:
This doc is not needed in the `get started` section. It should be there in `concepts`